assets/javascript/petapp.js

```diff
@@ -276,7 +271,7 @@ $(document).ready(function() {
   // this retrieves business data
   function getBizData() {
     var search = $("#selection-input").val();
-    var location = $("#address-input").val();
+    var location = $("#pac-input").val();
```

Comment:
Resolving - sounds like "pac-input" is a specification for Google Maps API functionality, so we'll keep it for now.